events/index.html

```diff
@@ -57,79 +58,40 @@ <h1 align="center" class="content-header">Past Events</h1>
-              PMs who will be sharing about their role and experience at HubSpot. Food will be provided.</p>
-          </div>
-        </div>
-        <div class="event-card">
-          <img src="../assets/events/20220908-board-game-night.PNG" />
-          <div class="event-card-header">Board Game Night</div>
-          <div class="event-card-subheader">09/08/2022</div>
-
-          <div class="event-card-body">
-            <p class="event-card-text">Join WCS for a fun night of board games and cookies! We will start off with speed
-              friending so you can get to know some new people and then we have a variety of board games for you to
-              choose from!</p>
-          </div>
-        </div>
-        <div class="event-card">
-          <img src="../assets/events/20220831-involvement-fair.png" />
-          <div class="event-card-header">Fall 22 Involvement Fair</div>
-          <div class="event-card-subheader">08/31/2022</div>
-          <div class="event-card-link"><a
-              href="https://docs.google.com/presentation/d/1a82TINeVAdjZ9CGUqA_Mjsq885jXdTg-vqOEeIIM0c8/edit?usp=sharing"
-              target="_blank">Slides</a>
-          </div>
-          <div class="event-card-body">
-            <p class="event-card-text">Join us to learn more about WCS' 5 committees, what we do for the broader CS
-              community and our members, and how you can get more involved. Here from our committee chairs, on the
-              roles, responsibilities, and application process, and get a chance to meet and get our questions answered
-              by our committee chairs!</p>
-          </div>
-        </div>
-        <div class="event-card">
-          <video src="../assets/events/20220829-gm.mp4" controls></video>
-          <div class="event-card-header">August General Meeting</div>
-          <div class="event-card-subheader">08/29/2022</div>
-          <div class="event-card-link"><a
-              href="https://docs.google.com/presentation/d/1jZVRfM1gsCfo1vGSSOTZSNASgerd8D27OTd0YJk6pfs/edit?usp=sharing"
-              target="_blank">Slides</a>
-          </div>
-          <div class="event-card-body">
-            <p class="event-card-text">Join us for WCS's first general meeting! We'll be going over some of our
-              announcements, progress, committees, and more from the summer and since the start of the school year! Come
-              by and learn how you can be a member of WCS.</p>
+      <script type="text/x-mustache" id="eventTemplate">
+        {{#event-admin}}
+        <div class="event-wrapper" id="allEvents">
+          <div class="event-card">
+            {{#video name}}
+              <video src="../assets/events/{{video name}}" controls></video>
+            {{/video name}}
+
+            {{#event image}}
+              <img src="../assets/events/{{event image}}" />
+            {{/event image}}
+
```

Comment:
run "format selection as HTML" on all of the files you push so that the formatting is consistent